OktaLogger.podspec

```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.homepage         = "https://github.com/okta/okta-logger-swift"
   s.license          = { :type => 'APACHE2', :file => 'LICENSE' }
   s.author           = { "Okta Developers" => "developer@okta.com" }
-  s.source           = { :git => "https://github.com/okta/okta-logger-swift.git",  :tag => s.version.to_s }
+  s.source           = { :git => "https://github.com/okta/okta-logger-swift.git",  :tag => "OktaLogger-"+s.version.to_s }
```

Comment:
I don't see any tags with the `OktaLogger-` prefix, you can leave this podspec as is

Reply:
yes makes sense, just make sure to create a new tag with the `OktaLogger-` prefix, otherwise it won't find it